vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/AbstractComboBoxIT.java

```diff
@@ -105,8 +105,7 @@ protected void assertNotRendered(String innerHTML) {
     }
```

Comment:
Note: Most of the changes in this PR are related to various integration tests that assume there to be a `<flow-component-renderer>` element.